addons/camellia-redis-proxy/values.yaml

```diff
@@ -5,7 +5,7 @@
 image:
   # refer: https://github.com/caojiajun/camellia-jdk21-bootstraps/blob/main/docs/redis-proxy/Dockerfile
   # this is the image of camellia-redis-proxy, which is built from Dockerfile of Apecloud/camellia-redis-proxy forked from caojiajun/camellia-jdk21-bootstraps.
-  registry: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com
+  registry: docker.io
   repository: apecloud/camellia-redis-proxy
```

Comment:
the namespace is "apecloud"

Reply:
The comments for this image have been added